crates/ruff_python_formatter/src/expression/expr_named_expr.rs

```diff
@@ -17,16 +18,37 @@ impl FormatNodeRule<ExprNamedExpr> for FormatExprNamedExpr {
             value,
             range: _,
         } = item;
-        write!(
-            f,
-            [
-                target.format(),
-                space(),
-                text(":="),
-                space(),
-                value.format(),
-            ]
-        )
+
+        write!(f, [target.format()])?;
+
+        let comments = f.context().comments().clone();
+        let trailing_target_comments = comments.trailing_comments(target.as_ref());
+        let leading_value_comments = comments.leading_comments(value.as_ref());
+        let dangling_item_comments = comments.dangling_comments(item);
+
+        if trailing_target_comments.is_empty() {
+            write!(f, [space()])?;
+        } else {
+            write!(f, [soft_line_break()])?;
```

Comment:
as opposed to?

Reply:
a `hard_line_break`, since as far as i can tell we always want to break after the comments if there are any. I realized it  doesn't make a difference here because a trailing comment will expand the group, but it might be easier to read